lua/dapui/render/loop.lua

```diff
@@ -54,9 +54,22 @@ function M.register_buffer(element_name, buf)
       .. buf
       .. ")"
   )
+  vim.cmd("augroup DAPUIExpandLongLines" .. buf)
+  vim.cmd(
```

Comment:
All of these autocmds should be created by the canvas module, not by the loop. Move this to the `render_buffer` function in canvas.lua

Reply:
Initially they was there but I placed in in wrong place https://github.com/rcarriga/nvim-dap-ui/issues/92#issuecomment-1093275771 so I moved it temporarily to any working place. Now autocommands are correctly created in `render_buffer`.